### PR TITLE
honor feature gates during bootstrapping

### PR DESCRIPTION
**Pull request by deads2k**
like https://github.com/openshift/cluster-kube-apiserver-operator/pull/1419 for kcm.  this one is also on the path to PSA enforcement via featuregate.

### CODE REVIEW

File: pkg/cmd/render/render.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package render
 import (
 	"errors"
 	"fmt"
+	configv1 "github.com/openshift/api/config/v1"
```
Comment (stlaz) [inline pkg/cmd/render/render.go:6]: move outside stdlib imports

**Comment by deads2k** [comment]
merge after https://github.com/openshift/cluster-kube-controller-manager-operator/pull/685

**Comment by deads2k** [comment]
tagging per slack, holding so the kube-update blocking PR can merge first.

**Pull request merged**